tests/Collector/SilverStripeCollectorTest.php

```diff
@@ -167,16 +180,18 @@ public function testGetWidgets()
         );
 
         $this->assertSame($expected, $result);
+        $this->logOut();
```

Comment:
`tearDown` should be doing logouts, so this shouldn't be needed

Reply:
Only in the `setUp()` method of `FunctionalTest` - so as I will set them back to `SapphireTest` I'll leave this in there

Reply:
ok